geometry_msgs/PoseStamped pose_stamped
---
uint32 seq # ack